src/resources/projects/website/listing/item-grid.ejs.md

```diff
@@ -37,83 +37,147 @@ return !["title", "image", "image-alt", "date", "author", "subtitle", "descripti
 
-<div class="card-body post-contents">
-<% if (showField('title')) { %><h5 class="no-anchor card-title listing-title"><%= item.title %></h5><% } %>
-<% if (showField('subtitle')) { %><div class="card-subtitle listing-subtitle"><%= item.subtitle %></div><% } %>
-<% if (showField('reading-time')) { %><div class="listing-reading-time card-text text-muted"><%= item['reading-time'] %></div> <% } %>
+::: {.card-body .post-contents}
+
+<% if (showField('title')) { %>
+<h5 class="no-anchor card-title listing-title"><%= item.title %></h5>
+<% } %>
+
+<% if (showField('subtitle')) { %>
+<div class="card-subtitle listing-subtitle"><%= item.subtitle %></div>
+<% } %>
+
+<% if (showField('reading-time')) { %>
+
+```{=html}
+<div class="listing-reading-time card-text text-muted"><%= item['reading-time'] %></div>
+```
+
+<% } %>
 
 <% if (fields.includes('categories') && item.categories) { %>
 
+```{=html}
 <div class="listing-categories">
-  <% for (const category of item.categories) { %>
-<div class="listing-category" onclick="window.quartoListingCategory('<%=utils.b64encode(category)%>'); return false;"><%= category %></div>
-  <% } %>
+<% for (const category of item.categories) { %>
+<div class="listing-category" onclick="window.quartoListingCategory('<%= utils.b64encode(category ) %>'); return false;"><%= category %></div>
+<% } %>
 </div>
+```
 
 <% } %>
+
 <% if (showField('description')) { %>
 
-<div class="card-text listing-description delink"><%= item.description %></div>
+```{=html}
+<div class="card-text listing-description delink">
+```
+
+<%= item.description %>
+
+```{=html}
+</div>
+```
+
 <% } %>
-<% 
+
+<%
 const flexJustify = showField('author') && showField('date') ? "justify" : showField('author') ? "start" : "end";
 %>
+
 <% if (showField('author') || showField('date')) { %>
-<div class="card-attribution card-text-small <%-flexJustify%>">
-<% if (showField('author')) { %><div class="listing-author"><%= item.author %></div><% } %>
-<% if (showField('date')) { %><div class="listing-date"><%= item.date %></div><% } %>
+
+```{=html}
+<div class="card-attribution card-text-small <%- flexJustify %>">
+```
+
+<% if (showField('author')) { %>
+<div class="listing-author"><%= item.author %></div>
+<% } %>
+
+<% if (showField('date')) { %>
+<div class="listing-date"><%= item.date %></div>
+<% } %>
+
+```{=html}
 </div>
+```
+
 <% } %>
 
 <% if (otherFields.length > 0) { %>
 
+```{=html}
 <table class="card-other-values">
-<% for (const field of otherFields) { 
-let value = readField(item, field);  
+<% for (const field of otherFields) {
+let value = readField(item, field);
 %>
 <tr>
 <td><%= listing.utilities.fieldName(field) %></td>
-<td class="<%-field%>"><%= listing.utilities.outputLink(item, field, value) %></td>
+<td class="<%- field %>"><%= listing.utilities.outputLink(item, field, value) %></td>
 </tr>
 <% } %>
 </table>
+```
```

Comment:
No markdown supports in metadata with this.
Splitting this to allow markdown for `listing.utilities.fieldName(field)` and `listing.utilities.outputLink(item, field, value)` will lead to `<p>` being added.

Reply:
For consistency until there is a decision, I switched to using raw blocks except for the lines with the variables to allow markdown.

9c43d630bc7a34f3dcf7f7465380420b2890d030

This means, there is still inline HTML.